boards/arm/mimxrt685_evk/pinmux.c

```diff
@@ -1051,5 +1099,84 @@ static int mimxrt685_evk_pinmux_init(const struct device *dev)
 	return 0;
 }
 
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(pca9420), okay) && \
+	CONFIG_SDMMC_USDHC_PMIC_1_8_VOL
```

Comment:
Can this be a more generic label like `CONFIG_USE_PMIC_PCA9420`

Reply:
Yes, do you think we should have 2 KConfig values, or just make the configuration itself more generic?